File: src/main/java/com/travel/domain/travel/dto/city/TravelCityDto.java
package com.travel.domain.travel.dto.city;

import com.travel.domain.travel.domain.city.TravelCity;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

@NoArgsConstructor
@Getter
public class TravelCityDto {
    private Long travelId;
    private Long cityId;

    @Builder
    public TravelCityDto(Long travelId, Long cityId) {
        this.travelId = travelId;
        this.cityId = cityId;
    }

    public TravelCity toEntity() {
        return TravelCity.builder()
                .travelId(travelId)
                .cityId(cityId)
                .build();
    }
}

File: src/main/kotlin/com/travel/domain/travel/dto/city/TravelCityDto.kt
package com.travel.domain.travel.dto.city

import com.travel.domain.travel.domain.city.TravelCity

data class TravelCityDto(
        val travelId: Long,
        val cityId: Long,
        val orderNo: Int = 1
) {
    fun toEntity(): TravelCity {
        return TravelCity(
                travelId = travelId,
                cityId = cityId,
                orderNo = orderNo
        )
    }
}